index.html

```diff
@@ -30,7 +31,17 @@
             </li>
           </ul>
         </nav>
-
+        <ul class="desk-menu">
+          <li>
+              <a href="#Portfolio" class="navnav-item" id="portfolio-link">Portfolio</a>
+          </li>
+          <li>
+              <a href="#About" id="about-link">About</a>
+          </li>
+          <li>
+              <a href="#Contact" id="contact-link">Contact</a>
+          </li>
+      </ul>
```

Comment:
- [ ] Great job so far but I think you didn't close the menu after clicking on the links as requested bu the privous code reviewer let me explain more when the user click on the link it will take him to the section that's good but the second part the menu should be closed aftr that but I can see it there even after clicking on the link and taking me to the section it is there so kindly make it disapear after clicking on any links 💯🔥🤩


https://user-images.githubusercontent.com/58553711/223397135-4615b611-fa5d-49a5-82f7-1611a666e4db.mp4